cloud-stream-custom-queue-names/src/main/java/com/solace/samples/spring/scs/GeneratedQueueNames.java

```diff
@@ -0,0 +1,37 @@
+package com.solace.samples.spring.scs;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.integration.StaticMessageHeaderAccessor;
+import org.springframework.integration.acks.AckUtils;
+import org.springframework.integration.acks.AcknowledgmentCallback;
+import org.springframework.messaging.Message;
+
+import com.solace.spring.cloud.stream.binder.util.SolaceAcknowledgmentException;
+
+import java.util.function.Consumer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@SpringBootApplication
+public class GeneratedQueueNames {
```

Comment:
Considering changing class name to CustomQueueNames

Reply:
Yep, will do.